BlueShift-iOS-SDK/BlueShiftUserInfo.m

```diff
@@ -152,30 +127,30 @@ + (BlueShiftUserInfo *)parseUserInfoDictionary:(NSDictionary *)currentUserInfoDi
     BlueShiftUserInfo *blueShiftUserInfo = nil;
     if (currentUserInfoDictionary) {
         blueShiftUserInfo = [[BlueShiftUserInfo alloc] init];
-        blueShiftUserInfo.email = [currentUserInfoDictionary objectForKey:@"email"];
-        blueShiftUserInfo.retailerCustomerID = [currentUserInfoDictionary objectForKey:@"customer_id"];
-        blueShiftUserInfo.name = [currentUserInfoDictionary objectForKey:@"name"];
-        blueShiftUserInfo.firstName = [currentUserInfoDictionary objectForKey:@"firstname"];
-        blueShiftUserInfo.lastName = [currentUserInfoDictionary objectForKey:@"lastname"];
-        blueShiftUserInfo.education = [currentUserInfoDictionary objectForKey:@"education"];
-        blueShiftUserInfo.facebookID = [currentUserInfoDictionary objectForKey:@"facebook_id"];
-        blueShiftUserInfo.gender = [currentUserInfoDictionary objectForKey:@"gender"];
-        if([currentUserInfoDictionary objectForKey:@"unsubscribed_push"] && [[currentUserInfoDictionary objectForKey:@"unsubscribed_push"] boolValue]) {
-            blueShiftUserInfo.unsubscribed = [[currentUserInfoDictionary objectForKey:@"unsubscribed_push"] boolValue];
+        blueShiftUserInfo.email = [currentUserInfoDictionary objectForKey:kEmail];
+        blueShiftUserInfo.retailerCustomerID = [currentUserInfoDictionary objectForKey:kBSUserCustomerId];
+        blueShiftUserInfo.name = [currentUserInfoDictionary objectForKey:kBSUserName];
+        blueShiftUserInfo.firstName = [currentUserInfoDictionary objectForKey:kBSUserFirstName];
+        blueShiftUserInfo.lastName = [currentUserInfoDictionary objectForKey:kBSUserLastName];
+        blueShiftUserInfo.education = [currentUserInfoDictionary objectForKey:kBSUserEducation];
+        blueShiftUserInfo.facebookID = [currentUserInfoDictionary objectForKey:kBSUserFacebookId];
+        blueShiftUserInfo.gender = [currentUserInfoDictionary objectForKey:kBSUserGender];
+        if([currentUserInfoDictionary objectForKey:kBSUserUnsubscribedPush]) {
+            blueShiftUserInfo.unsubscribed = [[currentUserInfoDictionary objectForKey:kBSUserUnsubscribedPush] boolValue];
         }
```

Comment:
when the stored value is `nil` should we set some default value here? Ex; `true`